features/step_definitions/claim_messages_steps.rb

```diff
@@ -19,6 +19,8 @@
 end
 
 Then(/^I should see my message at the bottom of the message list$/) do
+  sleep 1
```

Comment:
why is there a sleep in this step? should it not be wait_for_ajax or is it even needed if the page is actually refreshed?

I just think that this step could potentially cause flickers if the content is not available once it continues.